modules/migration-compatibility-guidelines.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+[id="migration-compatibility-guidelines_{context}"]
+= Compatibility guidelines
+
+You must install the {mtc-full} ({mtc-short}) version that is compatible with your {product-title} version.
+
+[cols="1,1", options="header"]
+.{product-title} and {mtc-short} compatibility
+|===
+|{product-title} version |{mtc-short} version^[1]^
+|3.7 to 4.5 |1.5.x
+|4.6 or later |1.6.x
+|===
+1. Install the latest z-stream version.
+
+You can migrate workloads from a source cluster with {mtc-short} 1.5.x installed to a target cluster with {mtc-short} 1.6.x installed as long as the `MigrationController` CR and the web console are running on the target cluster.
```

Comment:
Since this is the first mention of a custom resource in this module, I'd recommend using its full term: `MigrationController` CR -> `MigrationController` custom resource.

Reply:
Good catch. Will do.